core/frontend/src/internal/tile/IModelTile.ts

```diff
@@ -108,6 +108,14 @@ export class IModelTile extends Tile {
       return content;
 
     const sizeMultiplier = this.hasSizeMultiplier ? this.sizeMultiplier : undefined;
+
+    let ecefTransform: Transform;
+    try {
+      ecefTransform = this.tree.iModel.getEcefTransform();
+    } catch {
+      ecefTransform = Transform.createIdentity();
+    }
+
```

Comment:
```suggestion
    const ecefTransform = this.tree.iModel.isGeoLocated ? this.tree.iModel.getEcefTransform() : Transform.createIdentity();
```
Just check the condition that ensures it won't throw, instead of catching a predictable exception.